src/components/base.css

```diff
@@ -22,51 +43,59 @@ img {
 
 h1,
 h2,
-h3 {
-  font-size: 2em;
-  font-weight: normal;
+h3,
+h4,
+h5,
+h6 {
+  font-weight: var(--semibold);
+  line-height: var(--dense);
 }
 
 a {
   color: currentColor;
+  text-decoration: none;
 }
 
-.wrapper {
-  width: calc(100% - 10vmin);
-  margin: 0 auto;
-  padding: 5vmin 0;
+a:hover {
+  color: var(--primary);
 }
 
-/**
- * article grid
- */
-.article-list {
-  margin: 0;
-  padding: 0;
-  list-style: none;
-  display: grid;
-  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
-  grid-gap: 5vmin;
+a.active {
+  font-weight: var(--bold);
 }
 
-/**
- *
- */
-.section-headline {
-  padding: 0 0 0.4em 0;
-  margin: 0 0 5vmin 0;
-  border-bottom: 1px solid #ddd;
+tt,
+code,
+kbd {
+  background: var(--black-fade-5);
+  padding-bottom: 0.2em;
+  padding-top: 0.2em;
 }
 
-/**
- *
- */
-.list-inline {
-  margin: 0;
-  padding: 0;
-  list-style: none;
+tt,
+code,
+kbd {
+  font-family: var(--font-monospace);
+  font-size: 90%;
+}
+
+tt,
+code {
+  background-color: var(--black-fade-5);
+  border-radius: var(--radius-sm);
+  font-weight: var(--medium);
+  padding: 0.2em 0;
 }
 
-  .list-inline li {
-    display: inline-block;
-  }
+code:before,
+code:after,
+tt:before,
+tt:after {
+  content: ' ';
+  letter-spacing: -0.2em;
+}
+
+pre,
+code {
+  font-family: var(--font-monospace);
+}
```

Comment:
🧹 – `type` selectors only now. Maybe rename to `global.css`?

Reply:
Your call.

Reply:
Done in 65f5a25.